Watchman.Discord/Areas/Initialization/Services/ResponsesInitService.cs

```diff
@@ -23,28 +23,36 @@ public ResponsesInitService(ICommandBus commandBus, ResponsesGetterService respo
         public async Task InitNewResponsesFromResources()
         {
             var defaultResponses = this._responsesGetterService.GetResponsesFromResources();
-            var responsesInBase = this._responsesGetterService.GetResponsesFromBase().Select(response =>
+            
+            var responsesToUpdate = this._responsesGetterService.GetResponsesFromBase().Select(baseResponse =>
             {
                 // Update every response in the DB with availableVariables from defaultResponses
-                return new Response(response.OnEvent, response.Message, response.ServerId, 
-                    defaultResponses.First(responseNew => responseNew.OnEvent == response.OnEvent).AvailableVariables);
-            });
+                var matchingDefaultResponse = defaultResponses
+                    .FirstOrDefault(defaultResponse => defaultResponse.OnEvent == baseResponse.OnEvent);
```

Comment:
przenieś do jednej linijki ten FirstOrDefault
jak jest tylko jedno wyrażenie linq to nie ma sensu robić entera
jak jest ich więcej to wtedy to poprawia czytelność